shared/actions/config/index.js

```diff
@@ -390,6 +391,23 @@ const updateServerConfig = (state: TypedState) =>
     }
   })
 
+const writeLastSentXLM = (_: any, action: SetLastSentXLMPayload) => {
```

Comment:
the value is in the state already, just do
```
const writeLastSentXLM = (state) => {
...
value: {isNull:false, b: state.wallet.lastSentXLM}
```

Reply:
so you are saying to just get rid of writeFile and just write it every time? I was following the pattern of other stuff saved in app-state.json. Without the writeFile marker, it will re-write what it reads out on startup - maybe no big deal.